commit 325ac225f0aed800dc61f5b4670e86bdb043b40a (refs/changes/82/3182/4)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2018-01-11T00:32:10+00:00 (1 year, 9 months ago)
    
    MANTA-3546 lab deployments are created with too many vnodes
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/bin/manta-deploy-dev b/bin/manta-deploy-dev
index adfb379..4e5615c 100755
--- a/bin/manta-deploy-dev
+++ b/bin/manta-deploy-dev
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright (c) 2014, Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
 #
@@ -62,7 +62,7 @@ case "$1" in
 		;;
 	lab)
 		mdl_genconfig="lab"
-		mdl_topo_size=10000000
+		mdl_topo_size=131072
 		mdl_index_shards="1.moray 2.moray"
 		;;
 	*)
